AssetManagementTestProject/TestCases/LoginTest.cs

```diff
@@ -7,27 +7,49 @@ namespace AssetManagementTestProject;
 [TestFixture]
 public class LoginTest : NUnitWebTestSetup
 {
+    protected ChangePassword1stTimePage ChangePw1stTime;
+
+    [SetUp]
+    public void SetUp()
+    {
+        ChangePw1stTime = new ChangePassword1stTimePage();
+    }
     [Test]
     public void UserLoginSuccess()
     {
-        LoginPage.Login(Constant.ADMIN_USERNAME_HN, Constant.BASE_ADMIN_PASSWORD);
-        Asserter.AssertStringEquals(HomePage.ReturnHomePageUrl(), Constant.BASE_URL);
+        LoginPage.Login(Constant.ADMIN_USERNAME_HN, Constant.ADMIN_PASSWORD_HN);
+        DriverBaseAction.WaitToBeVisible(HomePage.ReturnHeaderHomePage());
+        DriverBaseAction.CompareUrls(Constant.BASE_URL);
     }
     [Test]
     public void UserLoginFirstTime()
     {
-        LoginPage.Login(Constant.STAFF_USERNAME_1, Constant.BASE_STAFF_PASSWORD_1);
+        LoginPage.Login(Constant.STAFF_USERNAME_1ST_TIME, Constant.STAFF_PASSWORD_1ST_TIME);
         DriverBaseAction.IsElementDisplayed(ChangePw1stTime.AskChangePwFirstLogin());
         DriverBaseAction.CompareUrls(ChangePw1stTime.ReturnExpectedChangePw1stTimeUrl());
```

Comment:
CompareUrls means that you're comparing 2 values. It does not matter even if they're different or not.
=> Do verification instead => Change the method name

Reply:
I am not quite clear about this. Do you mean changing the method name CompareUrls to something else (which means I have to adjust the core fw) or using fluent assertion to verify?

Reply:
Done. I created an asserter to assert two urls